contents/swift/welcome-to-swift.md

```diff
@@ -0,0 +1,144 @@
+# Swift
+
+Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md)
+
+# Overview
+
+Welcome to Swift, THE preferred language to do iOS Programming. Along with the awesome syntactic sugar and the playground which allows you to play and view results of your code, Here are some cool features that you might not be familiar with but would defnitely be useful to you if you are new to Swift. 
```

Comment:
If this book chapter is for people who is new to swift. They won't understand `syntactic sugar` or `playground` in swift. I would suggest you mention sth like swift is concise, easy to use and easy to learn. 

After you have an introduction of swift (like after a bunch of tutorial links), you can mention these as some cool features.

In this way, newbies can just refer to the section for tutorials. People who have a rough idea of what Swift is could refer to the cool features.